src/main/java/com/training/utilities/JsonReaderUtility.java

```diff
@@ -0,0 +1,32 @@
+package com.training.utilities;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Map;
+
+public class JsonReaderUtility {
+
+    public Map getJsonHashMap() {
+
+        JsonObject jsonObject = new JsonObject();
+        File jsonFile = new File("src/test/java/resources/testData.json");
+
+        try {
+            JsonElement fileElement = JsonParser.parseReader(new FileReader(jsonFile));
+            jsonObject = fileElement.getAsJsonObject();
```

Comment:
combine into one statement to avoid the use of declaring `JsonElement fileElement`